examples/sunpy_other_packages/reprojection_aia_euvi_mosaic.py

```diff
@@ -48,6 +48,12 @@
 
 maps = sunpy.map.Map(sorted(files))
 
+######################################################################
+# To reduce memory consumption we also downsample these maps before continuing,
+# you can disable this.
+
+maps = [m.resample((1024, 1024)*u.pix) for m in maps]
```

Comment:
Is it this just this examples we need to do this for?

Reply:
The main mem issues here seem to be inside reproject's coadd function so I hope it's only this one.